flink-runtime/src/test/java/org/apache/flink/runtime/security/token/hadoop/KerberosLoginProviderITCase.java

```diff
@@ -57,7 +58,7 @@ public void isLoginPossibleMustReturnFalseByDefault() throws IOException {
             UserGroupInformation userGroupInformation = mock(UserGroupInformation.class);
             ugi.when(UserGroupInformation::getCurrentUser).thenReturn(userGroupInformation);
 
-            assertFalse(kerberosLoginProvider.isLoginPossible());
+            assertFalse(kerberosLoginProvider.isLoginPossible(true));
```

Comment:
Such original tests are testing the no proxy cases. I would either set the new boolean on all non-proxy cases to false or test both proxy and non-proxy cases (true + false too) to be 100% sure that the original behavior is covered. It's your choice. Since we're under time pressure I would vote on the first.

Reply:
Addressed it.